lua/autorun/client/cl_focus.lua

```diff
@@ -0,0 +1,26 @@
+local tabbedOutList = {}
+local isTabbedOut = false
+
+timer.Create( "CFC_AttentionMonitor_tabNetTimmer", 1, 0, function()
+	local hasFocus = system.HasFocus()
+
+	if isTabbedOut == hasFocus then return end
+
+	net.Start( "CFC_AttentionMonitor_gameHasFocus" )
+		net.WriteBool( hasFocus )
+	net.SendToServer()
+	isTabbedOut = hasFocus
+end )
+
+net.Receive( "CFC_AttentionMonitor_sendData", function() -- receives the players that tabbed out
+	tabbedOutList = net.ReadTable()
+end )
+
+hook.Add( "PostDrawTranslucentRenderables", "afkRenderElements", function()
+	for k, _ in pairs( tabbedOutList ) do -- draws the icon for each tabbed out player

```

Comment:
instead of `k, _` just use `k`

Reply:
it might also be more clear to use something like `ply` instead of `k` , makes code easier readable.